src/ir_Airwell.cpp

```diff
@@ -16,18 +17,12 @@ const uint16_t kAirwellHdrSpace = 4 * kAirwellHalfClockPeriod;  // uSeconds
 const uint16_t kAirwellFooterMark = 5 * kAirwellHalfClockPeriod;  // uSeconds
 
 #if SEND_AIRWELL
-// Send an Airwell Manchester Code formatted message.
-//
-// Args:
-//   data:   The message to be sent.
-//   nbits:  The number of bits of the message to be sent.
-//           Typically kAirwellBits.
-//   repeat: The number of times the command is to be repeated.
-//
-// Status: BETA / Appears to be working.
-//
-// Ref:
-//  https://github.com/crankyoldgit/IRremoteESP8266/issues/1069
+/// Send an Airwell Manchester Code formatted message.
+/// Status: BETA / Appears to be working.
+/// @param[in] data The message to be sent.
+/// @param[in] nbits The number of bits of the message to be sent.
```

Comment:
reference to kAirwellBits has been removed ?
similar is true for a few other as well, I guess that is intentional.

Reply:
Yep. Correct.
Easier to cut & paste, and doxygen includes the "default" value a few lines above it in it's output.
I plan to leave it in the comments if there is something special, like multiple defaults or limits etc.